utils/src/ooinstall/cli_installer.py

```diff
@@ -871,7 +872,16 @@ def install(ctx, force):
     if len(oo_cfg.calc_missing_facts()) > 0:
         confirm_hosts_facts(oo_cfg, callback_facts)
 
+    # Write quick installer config file to disk:
     oo_cfg.save_to_disk()
+    # Write ansible inventory file to disk:
+    inventory_file = openshift_ansible.generate_inventory(hosts_to_run_on)
+
+    if gen_inventory:
+        click.echo()
+        click.echo("Generated ansible inventory: %s" % inventory_file)
+        click.echo()
+        sys.exit(0)
```

Comment:
Sounds good to me. I'll adjust tomorrow morn.

We should probably get it out of .ansible too someday.
On Mar 23, 2016 4:28 PM, "Jason DeTiberus" <notifications@github.com> wrote:

> In utils/src/ooinstall/cli_installer.py
> <https://github.com/openshift/openshift-ansible/pull/1654#discussion_r57222133>
> :
>
> >      oo_cfg.save_to_disk()
> > +    # Write ansible inventory file to disk:
> > +    inventory_file = openshift_ansible.generate_inventory(hosts_to_run_on)
> > +
> > +    if gen_inventory:
> > +        click.echo()
> > +        click.echo("Generated ansible inventory: %s" % inventory_file)
> > +        click.echo()
> > +        sys.exit(0)
>
> Any objections to displaying the path to the generated inventory for all
> deployments and just having the new flag only generate the inventory?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/1654/files/9c90c5a6071d2bcc9a4cab449327cc1d44f17726#r57222133>
>